kong/dynamic_hook/init.lua

```diff
@@ -213,6 +212,9 @@ end
 
 
 function _M.run_hooks(group_name, hook_name, a1, a2, a3, a4, a5, a6, a7, a8, ...)
```

Comment:
should we rename this to: `run_hook`? It looks like it's running only one

Reply:
done